python/tests/core/test_feature_view_engine.py

```diff
@@ -1977,7 +1977,7 @@ def test_check_feature_group_accessibility_get_type_python(self, mocker):
         # Assert
         assert (
             str(e_info.value)
-            == "Python kernel can only read from cached feature group. Please use `feature_view.create_training_data` instead."
+            == "Python kernel can only read from cached feature groups. When using external feature groups please use `feature_view.create_training_data` instead. If you are using spines, use a Spark Kernel."
```

Comment:
Do you have to use a Spark Kernel for spines?

Reply:
I don't think we can get it working with Hive. But we should be able to get it working with Flyingduck.

Reply:
https://arrow.apache.org/docs/python/generated/pyarrow.flight.FlightClient.html#pyarrow.flight.FlightClient.do_put

Reply:
only spark for now unfortunately